src/components/CertificateTemplates/tlds/sg/edu/smu/transcript-template/integration.spec.js

```diff
@@ -0,0 +1,43 @@
+import { Selector } from "testcafe";
+import { readFileSync } from "fs";
+import { join } from "path";
+import { getData } from "@govtechsg/open-attestation";
+
+fixture("Singapore Management University").page`http://localhost:3000`;
+
+const Certificate = "./transcript.opencert";
```

Comment:
i have further comments on the transcript.opencert contents but it cannot be opened on github due to its length
comments below:

1. key names should indicate what it contains, not arbitrary names and column numbers
2. contents of opencerts should only be data pertaining to the recipient's educational transcript, not display elements
3. why is there a date string in the name field?
4. empty fields should be omitted